bazel/scripts/defs.bzl

```diff
@@ -0,0 +1,46 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_context")
```

Comment:
meta: we have a directory "per feature" under the bazel directory. Everything under `bazel/scripts`.

All you have under `bazel/scripts` should go under a `bazel/lint`.